### PR TITLE
Rename calibration methods

### PR DESCRIPTION
**Pull request by mrwojtek**
After going through the reviews I propose those changes in names, they are more accurate in terms of what they do and they'll also not conflict with the future changes that are planned.

### CODE REVIEW

**Review by dstrain115 — APPROVED** [review]
Names are fine with me.  Will let other chime in as well.

**Review by balopat — APPROVED** [review]
LGTM!

File: cirq/google/__init__.py [hunk 1]
```diff
@@ -32,12 +32,12 @@
     PhasedFSimEngineSimulator,
     SQRT_ISWAP_PARAMETERS,
     THETA_ZETA_GAMMA_FLOQUET_PHASED_FSIM_CHARACTERIZATION,
-    make_floquet_request_for_circuit,
-    make_floquet_request_for_moment,
-    zeta_chi_gamma_calibration_for_moments,
-    run_characterizations,
+    prepare_floquet_characterization_for_circuit,
+    prepare_floquet_characterization_for_moment,
+    zeta_chi_gamma_compensation_for_moments,
```
Comment (mpharrigan) [inline cirq/google/__init__.py:37]: everything else has a verb

**Review by mpharrigan — APPROVED** [review]

File: cirq/google/__init__.py (hunk 1 above)
Comment (mrwojtek) [reply cirq/google/__init__.py]: That is true, I think make is the most appropriate (add could also work but only for this particular method, in future there might be more substantial circuit modifications).

**Pull request merged**